tuxemon/db.py

```diff
@@ -250,9 +263,29 @@ class MonsterEvolutionItemModel(BaseModel):
         ..., description="The monster slug that this evolution item applies to"
     )
     # optional fields
-    item: Optional[str] = Field(None, description="Item parameter.")
+    element: Optional[ElementType] = Field(
+        None, description="Element parameter"
+    )
     gender: Optional[GenderType] = Field(None, description="Gender parameter")
-    mixed: Optional[str] = Field(None, description="Mixed parameter.")
+    item: Optional[str] = Field(None, description="Item parameter.")
+    inside: bool = Field(
+        None,
+        description="Location parameter: inside true or inside false (outside).",
+    )
+    season: Optional[str] = Field(None, description="Season parameter.")
+    stat1: Optional[StatType] = Field(
+        None, description="Stat parameter stat1 >= stat2."
+    )
+    stat2: Optional[StatType] = Field(
+        None, description="Stat parameter stat2 < stat1."
+    )
+    tech: Optional[str] = Field(None, description="Technique parameter.")
+
+    @validator("tech")
+    def technique_exists(cls, v):
+        if has.db_entry("technique", v):
+            return v
+        raise ValueError(f"no resource exists in db: {v}")
```

Comment:
this error message is confusing

Reply:
I agree, blindly copy pasted the other one, I'm going to update the other as well.

Reply:
Changed a bunch of other resources too, let me know if it's ok for you

Reply:
Ah, I did black + isort as usual, and I included the small fix in prepare.py